### DISCUSSION TITLE
create_fts_index() not available

### DISCUSSION THREAD

**Discussion by AndiLeni** [lancedb/lancedb#1016] [Q&A]
I am a little bit confused about the `create_fts_index` function.
From the docs I am not sure if `create_index`  and `create_fts_index` are the same.
Could this be a mistake in the docs?

**Comment by wjones127** [comment]
There are different kinds of indices in LanceDB:

* Vector indices (made using `create_index`), which allow fast approximate nearest neighbors search (find similar vectors)
* Full text search indices (made using `create_fts_index`), which allows searching keywords in text
* Scalar indices (made using `create_scalar_index`), which allows searching for specific numbers or ranges of numbers in primitive columns (for example, look up a row by integer id)

Each of them have separate APIs to create them.

**Reply by AndiLeni** [reply]
Thanks for your answer. My IDE was not able to find the `create_fts_index ` function, but it seems to be there at runtime.
